src/tracker/face_id/identifier.py

```diff
@@ -166,9 +165,10 @@ def compute_known_embeddings(self):
                 LOGGER.debug(f"Added embedding for {directory}/{file}")
 
             if not embeddings:
-                raise NoFacesDetectedError(
+                LOGGER.warn(
                     f"Unable to recognize any face in pictures from "
                     f"{directory}. Are you sure it contains a picture of a "
                     f"person's face?"
                 )
-            self.known_embeddings[directory] = embeddings
+            else:
+                self.known_embeddings[directory] = embeddings
```

Comment:
If we change to a Warning, then what is the effect? Does it just skip the person from having any embeddings?

Reply:
no it doesn't skip the person, but the specific image for the person. The effect of the warning is that instead of entering a reconfig loop by raising an error, you just skip the image for this person, but you might be successful for another image of this same person.